packages/in_app_purchase/lib/src/billing_client_wrappers/purchase_wrapper.dart

```diff
@@ -89,11 +104,87 @@ class PurchaseWrapper {
   /// Note though that verifying a purchase locally is inherently insecure (see
   /// the article for more details).
   final String originalJson;
+
+  /// The payload specified by the developer when the purchase was acknowledged or consumed.
+  final String developerPayload;
+
+  /// Whether the purchase has been acknowledged.
```

Comment:
nit: I think it would be good to have a block here about what acknowledgement is and why it's important.

Reply:
added a brief explanation and linked to [BillingClient.acknowledgePurchase]